specification/ai/Azure.AI.Client/tspconfig.yaml

```diff
@@ -26,6 +26,12 @@ options:
     generate-test: false
     generate-sample: false
     examples-dir: "{project-root}/examples"
+  "@azure-tools/typespec-ts":
+    package-dir: "azure-ai-client"
+    isModularLibrary: true
+    packageDetails:
+      name: "@azure/ai-client"
+    flavor: azure
```

Comment:
You probably also want
 generate-test: false
 generate-sample: false
As we found no use for them in the other programming languages.